src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -277,6 +274,8 @@ public void configureCustomCA(CDAConfiguration configuration) throws InvalidConf
                 .initialRetryInterval(Duration.ofMillis(200)).maxAttempt(3)
                 .retryableExceptions(Collections.singletonList(ServiceUnavailableException.class)).build();
 
+        logger.info("Using private key at {} and certificate at {}", privateKeyUri, certificateUri);
```

Comment:
Nit: We have a structured logger. Use `kv` to set key value options rather than using string interpolation. This would become something like:

```
logger.atInfo().kv("privateKey", privateKeyUri).kv("certificate", certificateUri)
    .log("Configuring custom core CA");
```